app/controllers/prospects_controller.rb

```diff
@@ -0,0 +1,65 @@
+class ProspectsController < ApplicationController
+  skip_before_action :authenticate_user!
+  skip_before_action :check_no_institution!
+  skip_before_action :load_current_user_policies
+
+  def approve
+    Prospect.where(uuid: params[:id]).tap do |prospects|
+      if prospects.any?
+        invite(prospects.first)
+        redirect_to prospects_url,
+                    notice: "Successfully approved #{prospects.first.email}"
+      else
+        redirect_to prospects_url,
+                    warn: 'Could not find request to approve'
+      end
+    end
+  end
+
+  def create
+    @prospect = UserRequest.new(prospect_params)
+    if @prospect.valid?
+      @prospect.save!
+      redirect_to root_path, notice: I18n.t('access_request.submitted')
+    else
+      render :new
+    end
+  end
+
+  def index
+    @prospects = Prospect.pending
+  end
+
+  def new
+    @prospect = UserRequest.new
+  end
+
+  def reject
+    Prospect.where(uuid: params[:id]).tap do |prospects|
+      deny(prospects.first) if prospects.any?
+      redirect_to prospects_path
+    end
+  end
+
+  private
+
+  def deny(prospect)
+    prospect.update_attribute(:uuid, nil)
+  end
+
+  def invite(prospect)
+    User.invite!(email: prospect.email).tap do |user|
+      prospect.update_attribute(:uuid, nil)
```

Comment:
So after this line I need to create a policy to add the invited user to an institution. I guess I'll need to change the UI when I figure out how this works, but some help here for now would be appreciated.